azurelinuxagent/common/osutil/default.py

```diff
@@ -76,6 +79,31 @@ def __init__(self):
         self.selinux = None
         self.disable_route_warning = False
 
+    def get_firewall_dropped_packets(self, dst_ip=None):
```

Comment:
Is it possible to just track drops without enabling any drop actions?  I would like to gauge the telemetry before we enable the actual rules if possible.

Reply:
Not really. iptables attributes packets / bytes dropped / accepted to individual rules. The value the code reports is the number of dropped packets caused by *this* `DROP` rule. All prior values, since the rule did not exist, will be zero. And, since this is supposed to block inappropriate access, zero (or very nearly) seems a good baseline.